services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java

```diff
@@ -154,29 +154,29 @@ public Stream<MemberRepresentation> search(
         return provider.getMembersStream(organization, filters, exact, first, max).map(this::toRepresentation);
     }
 
-    @Path("{id}")
+    @Path("{memberId}")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @NoCache
     @Tag(name = KeycloakOpenAPI.Admin.Tags.ORGANIZATIONS)
     @Operation( summary = "Returns the member of the organization with the specified id", description = "Searches for a" +
             "user with the given id. If one is found, and is currently a member of the organization, returns it. Otherwise," +
             "an error response with status NOT_FOUND is returned")
-    public MemberRepresentation get(@PathParam("id") String id) {
+    public MemberRepresentation get(@PathParam("memberId") String id) {
```

Comment:
Nitpick:

```suggestion
    public MemberRepresentation get(@PathParam("memberId") String memberId) {
```

Reply:
Thanks. I did your suggestion and also updated the PR to use `member-id` as the path param. Noticed we are already using something like `user-id` in the User API so I updated it to follow the same format.